pkg/systemd/hwinfo.jsx

```diff
@@ -118,7 +118,7 @@ class SystemInfo extends React.Component {
                         { info.nproc !== undefined && <>
                             <DescriptionListGroup>
                                 <DescriptionListTerm>{ _("CPU") }</DescriptionListTerm>
-                                <DescriptionListDescription>{ (info.nproc > 1) ? `${info.nproc}x ${info.cpu_model}` : info.cpu_model }</DescriptionListDescription>
+                                <DescriptionListDescription>{ (info.nproc > 1) ? `${info.nproc}x ${info.cpu_model || _("unknown")}` : info.cpu_model }</DescriptionListDescription>
```

Comment:
Please fix this for the single-CPU case as well. I.e. pull this out into a new constant to avoid repetition.